build/elasticsearch/build.env

```diff
@@ -0,0 +1,26 @@
+WBS_ELASTICSEARCH_VERSION=1.0.0
+WBS_ELASTICSEARCH_VERSION_MAJOR=1
+WBS_ELASTICSEARCH_VERSION_MINOR=1.0
```

Comment:
There is no way around this? Adee had this nice shell thingy, but we are not using this in shell context anymore, right?

Reply:
We still have this set of variables. You mentioned something about that yesterday. Unfortunately I forgot about the way forward you mentioned.

Reply:
The version tags are no parsed in a similar way to before and part of the `build/build.sh` script which handles both building for test runs (both locally and on GitHub), and building and pushing to Dockerhub. The version tags are only applied when the build is ran from the `nx release publish` task...

Reply:
👍🏾